src/Webinex.Calendar/Repeats/Calculators/WeekdayRepeatEventCalculator.cs

```diff
@@ -81,7 +84,6 @@ private Period Map(RecurrentEvent @event, Occurrence x)
 
     private LocalDateTime Tz(DateTimeOffset value, string tz)
     {
-        var localDateTime = value.ToUtc().DateTime.ToLocalDateTime();
-        return localDateTime.InZoneLeniently(DateTimeZoneProviders.Tzdb[tz]).LocalDateTime;
+        return value.ToInstant().InZone(DateTimeZoneProviders.Tzdb[tz]).LocalDateTime;
```

Comment:
Actually, utc can be converted to zone without a problems with ambigues time and double time. So it's safe

Reply:
yes, any instant/global time could be converted to local time, but not all local time are ambitious/exists